docs/settings/canned-message-plugin.md

```diff
@@ -25,9 +25,12 @@ Once plugin settings are changed, a **reset** is required for them to take effec
 | :-----: | :---------------: | :-----: |
 | canned_message_plugin_allow_input_source | `rotEnc1`, `_any` | `_any` |
 | canned_message_plugin_enabled | `true`, `false` | `false` |
-| canned_message_plugin_messages | `string` | `""` |
 | canned_message_plugin_send_bell | `true`, `false` | `false` |
 
+Messages can be set with a dedicated option:
+
+`--set-canned-message "<messages>"`
```

Comment:
I'm thinking on how to work this in given that it doesn't use the `--set` command now. I'm wondering about putting it in the table at the bottom (alphabetized settings) like:
```
| messages | `string` | `""` |
```

I know it's not really a protobuf any more as it's been deprecated, but what do you think? It's still a vital "setting" to set for the plugin.

Reply:
Can be a table. I'm not sure how it will look.

Reply:
Yeah I'd just swap out `messages` for `canned_message_plugin_messages` on [this line](https://github.com/meshtastic/Meshtastic/blob/cfe349efccb81184d66cfb6cf55e58ab9e3c484c/docs/settings/canned-message-plugin.md?plain=1#L32) or put the actual protobuf name that the settings are being saved to instead of `messages`

Reply:
Also, once it's swapped I try to keep these tables alphabetical. So keep that in mind as well.